File: springboot/open-api/src/main/java/com/openapi/domain/dto/ws/response/TtsDataResponse.java
package com.openapi.domain.dto.ws.response;

import com.openapi.domain.dto.ws.base.StreamSeqDto;
import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper = true)
@Data
public class TtsDataResponse extends StreamSeqDto {

    /**
     * 与当前音频分片对应的字幕/文案（流式 TTS 可与播放同步展示）。
     */
    private String text;

    private String base64AudioStream;
}

File: app/android/app/src/main/java/com/magicvector/domain/dto/ws/response/TtsDataResponse.kt
package com.magicvector.domain.dto.ws.response

import com.magicvector.domain.dto.ws.base.StreamSeqDto

data class TtsDataResponse(
    val base64AudioStream: String,
    /** 与当前音频分片对应的字幕/文案（流式 TTS 可与播放同步展示） */
    val text: String = "",
) : StreamSeqDto()